.gitignore

```diff
@@ -19,20 +19,15 @@ goex/docker/misc/images.json
 
 ################## Berkley Function Call Leaderboard ##########################
 
-# Ignore tree-sitter
```

Comment:
Why not ignore the treesitter?

Reply:
It's outdated. We install treesitter through pip now.